### ISSUE TITLE
add Mouse support (in full screen)

### ISSUE THREAD

**Issue by menthorul** [linuxmint/hypnotix#262]
add Mouse support (in full screen) please, right click to exit full screen - ex.
thanks

**Comment by Axel-Erfurt** [comment]
This is not possible with embedded mpv.

**Comment by DYefremov** [comment]
Not at all. It's possible. For example, I have implemented through a double click. But... You may need to permanently disable mouse events and osc for MPV.
